### DISCUSSION TITLE
Adding Navbar in all files

### DISCUSSION THREAD

**Discussion by Karan-S-dev** [facebook/docusaurus#7802] [Q&A] [answered]
Hi, I am able to see navbar in lnding page nd all md pages. But I have some extra js files for login, Q&A and so on. In those md files, navbar is not visible. Is there any way i can add navbar in all pages of my documentation site?

**Comment by Josh-Cena** [comment, marked as answer]
You need to wrap your React pages with `<Layout>`. See https://docusaurus.io/docs/creating-pages#add-a-react-page

**Reply by Karan-S-dev** [reply]
Thank you so much. I was sitting in this problem for more than a week. You saved my time .
